python_modules/dagster/dagster/_core/storage/runs/schema.py

```diff
@@ -45,6 +45,8 @@
     db.Column("start_time", db.Float),
     db.Column("end_time", db.Float),
     db.Column("backfill_id", db.String(255)),
+    db.Column("run_creation_time", db.Float),
+    db.Column("public_update_timestamp", db.Float),
```

Comment:
If these new columns are going to power the run timeline going forward, we're going to need the equivalent of

```
db.Index(
    "idx_run_range",
    RunsTable.c.status,
    RunsTable.c.update_timestamp,
    RunsTable.c.create_timestamp,
    mysql_length={
        "status": 32,
        "update_timestamp": 8,
        "create_timestamp": 8,
    },
)
```